.github/workflows/composed-ci-management-verify.yaml

```diff
@@ -131,6 +131,8 @@ jobs:
       GERRIT_PATCHSET_REVISION: ${{ inputs.GERRIT_PATCHSET_REVISION }}
       GERRIT_PROJECT: ${{ inputs.GERRIT_PROJECT }}
       GERRIT_REFSPEC: ${{ inputs.GERRIT_REFSPEC }}
+      ENV_VARS: ${{ toJSON(vars) }}
+      ENV_SECRETS: ${{ toJSON(secrets) }}
```

Comment:
Given what actionlint is complaining about this should be moved to the secrets context below.

Reply:
Resolved